scraper/scraper.py

```diff
@@ -125,17 +124,15 @@ def export_data(centres_cherchés, outpath_format='data/output/{}.json'):
         else:
             logger.warning(f"le centre {centre['nom']} ({code_departement}) n'a pas pu être rattaché à un département connu")
 
-    outpath = outpath_format.format("info_centres")
-    with open(outpath, "w") as info_centres:
-        json.dump(par_departement, info_centres, indent=2)
+    outpath = Path(outpath_format.format("info_centres"))
+    outpath.write_text(json.dumps(par_departement, indent=2))
```

Comment:
Pareille ici, j'aurais tendance à préférer : 
```suggestion
        with outpath.open("r") as fichier_dispo:
            json.dump(disponibilités, fichier_dispo, indent=2)
```